ote/requirements.txt

```diff
@@ -2,4 +2,3 @@ requests
 pyyaml
 mmcv
 texttable
-pytest
```

Comment:
Could you do some useless change in pytorch_toolkit/tests/run_model_template_tests.sh like this https://github.com/openvinotoolkit/training_extensions/pull/460/commits/6dd4779b5c56f6282deda7a3956cee90231caad0
in order to run existing tests. CI has not been reconfigured properly yet. That's why you should do it.

I guess tests will be failed due to this change (removing `pytest`).

Reply:
We have separate requirements.txt, and now all req's for accuracy validation stored in tests/accuracy_validation/requirements.txt including pytest.

Reply:
I see, but pytest is needed here https://github.com/openvinotoolkit/training_extensions/blob/develop/tests/run_model_templates_tests.sh#L24

Reply:
Ok, got it, will update soon!